src/main/java/fr/moribus/imageonmap/commands/maptool/NewCommand.java

```diff
@@ -87,9 +88,31 @@ protected void run() throws CommandException {
         if (args.length < 1) {
             throwInvalidArgument(I.t("You must give an URL to take the image from."));
         }
-
+        //Checking if the map limit and image limit
+        if (!Permissions.BYPASS_IMAGE_LIMIT.grantedTo(player)) {
+            int imageLimit = Permissions.NEW.getLimitPermission(player, Permissions.LimitType.image);
+            int imageCount = MapManager.getPlayerMapStore(player.getUniqueId()).getImagesCount();
+            if (imageLimit <= imageCount) {
+                throwInvalidArgument(
+                        I.t("Your image limit is set to {0} and you currently have {1} ", imageLimit, imageCount));
```

Comment:
Consider: `and you currently have {1} image(s) active`

maybe loaded instead of active?

Reply:
I have choosed `Your image limit is set to {0} and you currently have {1} loaded image(s)` as well for the map limit that is `Your map limit is set to {0} and you currently have {1} loaded map(s)`